src/optimizer/statement_to_opr_convertor.py

```diff
@@ -225,6 +226,18 @@ def visit_load_data(self, statement: LoadDataStatement):
         load_data_opr = LogicalLoadData(table_metainfo, statement.path)
         self._plan = load_data_opr
 
+    def visit_upload(self, statement: UploadStatement):
+        """Convertor for parsed load data statement
```

Comment:
Please, change this doc string to reflect the Upload statement. Thanks!

Reply:
Fixed.